meilisearch-http/src/routes/indexes/search.rs

```diff
@@ -30,16 +29,16 @@ pub struct SearchQueryGet {
     q: Option<String>,
     offset: Option<usize>,
     limit: Option<usize>,
-    attributes_to_retrieve: Option<CS<StarOr<String>>>,
-    attributes_to_crop: Option<CS<StarOr<String>>>,
```

Comment:
So, it is no more possible to use the `StarOr` helper? 🥹

Reply:
The issue with this particular route is that its fields are moved into a really similar structure that handles the post route directly. And thus, it's the other one that takes the wildcard.

Reply:
Ho yeah, you are right, thank you for the explanation!